airflow/providers/google/cloud/sensors/gcs.py

```diff
@@ -99,10 +102,43 @@ def poke(self, context: Context) -> bool:
         )
         return hook.exists(self.bucket, self.object, self.retry)
 
+    def execute(self, context: Context) -> None:
+        """Airflow runs this method on the worker and defers using the trigger."""
+        if self.deferrable is False:
```

Comment:
```suggestion
        if not self.deferrable:
```

(Maybe the entire block can be flipped to `if self.deferrable ... else`?)

Reply:
I just changed it into `if not self.deferrable`. As the default value of `self.deferrable` is `False`, I think it might be better to keep it as it is.